LibCore/CConfigFile.cpp

```diff
@@ -112,19 +112,22 @@ Color CConfigFile::read_color_entry(const String& group, const String &key, Colo
         return default_value;
     }
 
-    Vector<String> shades = read_entry(group, key).split(',');
-    bool ok = shades.size() >= 3;
+    auto shades = read_entry(group, key).split(',');
+    if (shades.size() < 3)
+        return default_value;
+    bool ok1, ok2, ok3, ok4 = true;
```

Comment:
Only `ok4` is initialized here.
I prefer to only create one variable per line so that bugs like this become obvious :)

Reply:
It wasn't really a bug per-se. Only the ok4 *needs* to be initialized. I was being lazy.

Reply:
Or so I thought..